swarm/storage/chunker.go

```diff
@@ -298,6 +305,13 @@ func (self *TreeChunker) hashChunk(hasher SwarmHash, job *hashJob, chunkC chan *
 	job.parentWg.Done()
 
 	if chunkC != nil {
+		//NOTE: this increases the chunk count even if the local node already has this chunk;
+		//on file upload the node will increase this counter even if the same file has already been uploaded
+		//So it should be evaluated whether it is worth keeping this counter
```

Comment:
I would definitely find this misleading. This is probably one of the most crucial metrics, we should make sure it is as accurate as possible.

Reply:
Agreed. I expect many changes with the rewrite and the syncer rewrite, so I thought it's good we have the metric in the code now but will probably need to update later anyway, and wherever we move it now it is probably wrong.